acceptance/acceptance_test.go

```diff
@@ -38,6 +54,23 @@ var Scripts = map[string]bool{
 }
 
 func TestAccept(t *testing.T) {
+	testAccept(t, InprocessMode, "")
+}
+
+func TestInprocessMode(t *testing.T) {
+	if InprocessMode {
+		t.Skip("Already tested by TestAccept")
+	}
+	if runtime.GOOS == "windows" {
+		// -  catalogs                               A catalog is the first layer of Unity Catalog’s three-level namespace.
+		// +  catalogs                               A catalog is the first layer of Unity Catalog�s three-level namespace.
+		t.Skip("Fails on CI on unicode characters")
```

Comment:
Any idea why this happens? Could have something to do with JSON serialization/deserialization.

https://pkg.go.dev/encoding/json#Encoder.SetEscapeHTML